.travis.yml

```diff
@@ -7,6 +7,8 @@ python:
 
 env:
   - DJANGO="django>=1.8,<1.9"
+  - DJANGO="django>=1.9,<1.10"
+  - DJANGO="django>=1.10,<1.11"
```

Comment:
Missing master version.
Maybe convert it to tox-travis version eg. https://github.com/ArabellaTech/drf_tweaks/blob/master/.travis.yml